CHANGELOG.md

```diff
@@ -67,6 +67,21 @@ the following form: YYYY.0M.0D.
 - Created initial drafts of the six portfolio assessments
 - Added READMEs to key folders like `test` and `lib` to explain their purpose
 
+## [2024.09.15]
+
+### Added
+
+- Designed a `bank account` component
+- Designed a `music player` component
+- Designed a `shopping cart` component
+
+## [2024.10.02]
```

Comment:
Looks good! I tend to prefer reverse chronological order, so the newest changes are on top.